pkg/runctl/controller.go

```diff
@@ -515,6 +533,14 @@ func (c *Controller) handlePipelineRunPrepare(
 			return true, err
 		}
 
+		ctx = k8s.UpdateLoggerContext(ctx, "",
+			"runBuildNamespace", namespace,
+			"runBuildAuxNamespace", auxNamespace,
```

Comment:
Don't use the term "build".
```suggestion
			"pipelineRunExecutionNamespace", namespace,
			"pipelineRunExecutionAuxiliaryNamespace", auxNamespace,
```